menas/ui/service/EntityValidationService.js

```diff
@@ -52,7 +52,7 @@ var EntityValidationService = new function () {
     } else if (GenericService.hasWhitespace(oEntity.name)) {
       oInput.setValueState(sap.ui.core.ValueState.Error);
       oInput.setValueStateText(
-        sEntityType + " name '" + oEntity.name + "' should not have whitespaces. Please remove spaces and retry");
+        sEntityType + " name can only contain alphanumeric characters and underscores");
```

Comment:
Why removing the info of the offending entry (`oEntity.name`)?

Reply:
Because it tells you nothing. It is what you have written in the field above it.
and
This is the same solution as in Conformance Rules